charts/qat-device-plugin/README.md

```diff
@@ -0,0 +1,43 @@
+# Intel QAT Device Plugin Helm Chart
+
+## Get Helm Repository Info
+```
+helm repo add intel https://intel.github.io/helm-charts/
+helm repo update
+```
+You can then run helm search repo intel to see pulled charts.
```

Comment:
put the `helm search repo intel` into single quotes so it looks like a command.  Same for all the charts.